### PR TITLE
Remove gmp/mpir from the run section

### PR DESCRIPTION
**Pull request by bluescarni**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bluescarni** [comment]
@conda-forge-admin, please rerender

**Comment by bluescarni** [comment]
@isuruf Now that both gmp and mpir are using ``run_exports``, I think we can remove them from the run section here?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ source:
   - patches/corei5.patch
 
 build:
-  number: 0
+  number: 1
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you keep this at `0`? There's no need for a rebuild.

**Comment by bluescarni** [comment]
@conda-forge-admin, please rerender

**Comment by bluescarni** [comment]
@conda-forge-admin, please rerender

**Comment by bluescarni** [comment]
@isuruf is this looking good to go now?

**Comment by bluescarni** [comment]
@conda-forge-admin, please rerender

**Pull request merged**